### PR TITLE
Adding modal/autologin after register

### PR DESCRIPTION
**Pull request by AreYouIvan**
Hello team!

In this PR there's a modal that improves a lot the UX because this makes an auto log in, and then redirects to /home.

Additional I made other changes:
- Changed the initialize of localStorage, because this one, was initializing before the login and this makes a vulnerability
- Fixed the redirect problem in production
- Added the Twitter login button (Just the layout)
- I made a partial deploying to test all the app in production

You can test the web app [here](https://studio-ghibli-se.netlify.app/home)

image

### CODE REVIEW

File: src/pages/profile/index.js [hunk 1]
```diff
@@ -1,81 +1,108 @@
...
-            initial={linkAnimateFrom}
-            animate={linkAnimateTo}
-            transition={{delay: 0.10}}
-          >
-            <button type="button" onClick={()=> setSettingsOpen(!settingsOpen)}> Settings <img src={linkArrow}></img></button>
-          </motion.div>
-            { settingsOpen &&
+      <div className="profilePageContainer">
+        <header className="profile-header">
+          {" "}
```
Comment (Hai5edfm) [inline src/pages/profile/index.js:27]: what is this? :v

File: src/pages/profile/index.js [hunk 2]
```diff
@@ -1,81 +1,108 @@
...
+            <motion.div
+              initial={linkAnimateFrom}
+              animate={linkAnimateTo}
+              transition={{ delay: 0.1 }}
+            >
+              <button
+                type="button"
+                onClick={() => setSettingsOpen(!settingsOpen)}
+              >
+                {" "}
```
Comment (Hai5edfm) [inline src/pages/profile/index.js:51]: ?

**Review by Hai5edfm — APPROVED** [review]
looks good to me :v

**Review by LilianaRestrepoTorres — APPROVED** [review]

File: src/pages/profile/index.js (hunk 1 above)
Comment (AreYouIvan) [reply src/pages/profile/index.js]: I forgot to delete the brackets of the render conditional, but I already deleted them. thank you!

**Pull request merged**